wurst/systems/spells/mage/elementalist/OverCharge.wurst

```diff
@@ -0,0 +1,148 @@
+@configurable constant int DUMMY_AURA_ID = 'ESOA'
+@configurable constant int DUMMY_AURA_BUFF_INT = 'ESOB'
+@configurable constant string DUMMY_AURA_BUFF_STR = "ESOB"
+@configurable constant real COOLDOWN = 20
+@configurable constant int MANACOST = 15
+@configurable constant real DURATION = 10
+
+@configurable constant string SPELL_ICON = "ReplaceableTextures\\PassiveButtons\\PASBTNFeedBack.blp"
+@configurable constant string AURA_EFFECT_PATH = "Abilities\\Spells\\Human\\ManaFlare\\ManaFlareBase.mdl" //"Abilities\\Spells\\Human\\ManaShield\\ManaShieldCaster.mdl"
+@configurable constant string TOOLTIP_NORM = "|c00ffcc00A |r- Overcharge"
+@configurable constant string TOOLTIP_EXT = "Causes the next spell you cast to have a bonus effect.\n\n|c00ffcc00Za|r|c00ffcc00p|r|c00ffcc00|r|nForks to three targets.\n\n|c00ffcc00Meteo|r|c00ffcc00r|r|c00ffcc00|r|nSets the ground on fire dealing 15 damage per second for 7 seconds.\n\n|c00ffcc00Force Pus|r|c00ffcc00h|r|c00ffcc00|r|nIncreases knockback and stuns enemies.\n\n|c00ffcc00Earth Guardian|r|c00ffcc00s|r|c00ffcc00|r|nDoesnt require channeling to summon orbs.\n\n|c00ffcc00Meditat|r|c00ffcc00e|r|c00ffcc00|r|nDoesnt require standing still to regenerate. Cancels on damage."
+
+@compiletime function createBuffs()
+    new BuffDefinition(DUMMY_AURA_BUFF_INT, 'Bstt')
+    ..setArtTarget(1, "")
+    ..setTooltipNormal(1, "Overcharge")
+    ..setTooltipNormalExtended(1, "Next spellcast is empowered.")
+    ..setIcon("ReplaceableTextures\\PassiveButtons\\PASBTNFeedBack.blp")
+    ..setIconNormal(1, "ReplaceableTextures\\PassiveButtons\\PASBTNFeedBack.blp")
+    //..setArtTarget(1, AURA_EFFECT_PATH)
+    //..setTargetAttachmentPoint0(1, "hand, left")
+    //..setTargetAttachmentPoint1(1, "hand, right")
+    ..setTargetAttachments(1, 2)
+
+@compiletime function createSpell()
+/*
+    new AbilityDefinitionIllidanChannel(ABILITY_ID)
+    ..setTargetType(1, 0)
+    ..setArtCaster("")
+    ..setArtEffect("")
+    ..setArtTarget("")
+    ..setArtSpecial("")
+    ..setButtonPositionNormalX(0)
+    ..setButtonPositionNormalY(1)
+    ..setButtonPositionResearchX(0)
+    ..setButtonPositionResearchY(1)
+    ..setIconNormal(SPELL_ICON)
+    ..setIconResearch(SPELL_ICON)
+    ..setIconTurnOff(SPELL_ICON)
+    ..setHeroAbility(false)
+    ..setItemAbility(false)
+    ..setLevels(1)
+    ..setHotkeyNormal("A")
+    ..setName("Overcharge")
+    ..setDisableOtherAbilities(1, false)
+    ..setFollowThroughTime(1, DURATION)
+    ..setTooltipNormal(1, TOOLTIP_NORM)
+    ..setTooltipNormalExtended(1, TOOLTIP_EXT)
+    ..setBaseOrderID(1, "defend")
+    ..setOrderStringUseTurnOn("defend")
+    ..setOrderStringActivate("defend")
+    ..setOrderStringDeactivate("defend")
+    ..setOrderStringTurnOff("defend")
+    ..setOptions(1, 1)
+    ..setCooldown(1, COOLDOWN)
+    ..setManaCost(1, MANACOST)
+    ..setAnimationNames("")
+    ..setCastingTime(1, 0)
+
+    */
+    new AbilityDefinitionWindWalk(ABILITY_ID)
```

Comment:
Why wind walk?

Reply:
Casts without interrupting current order, no animation to have a smooth feel for when casting overcharge + following spell

Reply:
Got it, makes sense 👍